source/wp-content/themes/wporg-parent-2021/patterns/404-page.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: 404 Page
+ * Slug: wporg-parent-2021/404-page
+ * Categories: wporg
```

Comment:
This isn't a pattern meant to be used in content, so it doesn't need a category. Instead, we can disable it from the editor using:
```suggestion
 * Inserter: no
```